frontend/src/employee-mobile-frontend/components/messages/ThreadView.tsx

```diff
@@ -150,6 +150,9 @@ const RoundIconButton = styled.button`
 
 const MessageContainer = styled.div`
   border-radius: ${defaultMargins.s};
+  max-width: 90%;
+  word-break: break-word;
```

Comment:
You should probably use this only for `MessageContent` since otherwise it can for example cut the message times (happened to me locally).

Reply:
Ok thanks!

Also it seems that word-break: break-word; is deprecated: https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

Reply:
Yeah seems like that's the right call then!